anonlink/solving/_multiparty_solving_inner.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _multiparty_solving_inner_h
+#define _multiparty_solving_inner_h
+
+#include <vector>
+
+
+struct Record {
+    unsigned int dset_i;
+    unsigned int rec_i;
+    
+    Record() {}
+    Record(unsigned int dset_i_, unsigned int rec_i_) : dset_i(dset_i_), rec_i(rec_i_) {}
+    bool operator==(const Record &other) const {
+        return dset_i == other.dset_i && rec_i == other.rec_i;
+    }
+};
```

Comment:
Can you move this declaration to the `.cpp` file and just put `struct Record;` in the header file? Normally that would be fine, but maybe some generated Cython code needs the details?

Reply:
Unfortunately, Cython needs it.